llvm/include/llvm/migration_cmd_examples.inc

```diff
@@ -0,0 +1,123 @@
+#ifdef _WIN32
+const char *const DPCTExamplesMsg = R""""(
```

Comment:
what is the diff between two DPCTExamplesMsg? Avoid modifying 2 for each update. How about we String A  + String B + String C ;  (maybe String B is specific for windows).

Reply:
How about I maintain a single str with "#ifdef WIN32"?

Will that be okay?